### ISSUE TITLE
intltool and gettext

### ISSUE THREAD

**Issue by jjstickel** [Qalculate/libqalculate#47]
With intltool as part of the build, autoreconf needs to be rerun in a macports environment so that the correct perl is used. Now with the addition of gettext in 2.x, I get this error:
`config.status: error: po/Makefile.in.in was not created by intltoolize.
`
It seems that intltool and gettext cause problems when used together:
[https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=724555](url)

Can you fix or suggest a workaround? Thanks.

**Comment by hanna-kn** [comment]
Does it help if you run ./autogen.sh instead of autoreconf?

**Comment by jjstickel** [comment]
Yes, that worked, although I had to patch autogen.sh to use "glibtoolize" (gnu libtool) rather than "libtoolize". Thanks.
